tests/isar/apis/security/test_authentication.py

```diff
@@ -0,0 +1,69 @@
+import json
+from http import HTTPStatus
+
+import pytest
+from jose import jwt
+
+
+def mock_access_token():
+    claims = {
+        "iss": "ISAR",
+        "iat": 1637588872,
+        "exp": 1669124872,
+        "aud": "test_client@example.com",
+        "sub": "user_impersonation@example.com",
+    }
+
+    key = "qwertyuiopasdfghjklzxcvbnm123456"
+    token = jwt.encode(claims=claims, key=key)
```

Comment:
Is there a reason why the claims and key mock are this detailed? 

It should be sufficient with something like 

```
claims = {}
key = "some_key"
```